wof/wofpy_config.py

```diff
@@ -53,9 +60,18 @@ def copytree(src, dst, symlinks=False, ignore=None):
 def main():
     args = docopt(__doc__, version='0.1.0')
     directory = args.get('INDIR')
+    mode = args.get('--mode') 
+    if mode not in ['development', 'production']:
+        raise ValueError('Got mode: {!r}, expected development, or production.'.format(mode))
 
     makedirs(directory)
-    copytree(_CONFIG, directory)
+    _TEST = os.path.join(directory, 'odm2', 'timeseries')
```

Comment:
@lsetiawan and @emiliom I went ahead and modified this to reflect the source code instead of https://github.com/ODM2/WOFpy/issues/132#issuecomment-304358051